### ISSUE TITLE
When I first log in, I am logged in to a blank screen

### ISSUE THREAD

**Issue by tim-sendible** [argoproj/argo-workflows#5494]
## Summary 

When I first log in, or when a new user first logs in, I am presented with a blank page. I then have to choose one of the menu items in order to actually see anything.

As a new user this can be confusing/daunting.

I propose you send them to the workflows tab or something so that the UI isn't quite so sparse.

To prepare to reproduce this issue, go to your base URL and then log out. eg:

1. Go to argo-workflows.yourcoompany.biz/
2. Go to the User tab, and then log out.

1. On the /login page, log in.
2. This is what I see:

![image](https://user-images.githubusercontent.com/69461449/112110635-1866f080-8bab-11eb-926a-c7f9569f8106.png)

## Diagnostics

Some (hopefully) useful information for this one:

My base_href is set as follows:
```
        env:
          - name: BASE_HREF
            value: /
```

I authenticate using github sso via Argo CD

What Kubernetes provider are you using? EKS 1.19

What version of Argo Workflows are you running? 3.0 rc8

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by simster7** [comment]
@tim-sendible Is this an SSO login?

**Comment by simster7** [comment]
Or is it a "token" login?

**Comment by simster7** [comment]
Also if you could attach the URL that gives you the "blank" screen that could help debug

**Comment by tim-sendible** [comment]
It's sso, and the url is just my base url with no trailing slash.

**Comment by simster7** [comment]
Yup, just found the issue

**Comment by simster7** [comment]
This has been fixed in [`v3.0.0-rc9`](https://github.com/argoproj/argo-workflows/releases/tag/v3.0.0-rc9)
